environment.yml

```diff
@@ -3,73 +3,9 @@ channels:
   - conda-forge
 # - defaults
 dependencies:
-# 1/4 req main
-    - python>=3.8,<=3.10
-    - numpy
-    - pandas
-    - pip
-
-    - py-find-1st
-    - aiohttp
-    - SQLAlchemy
-    - python-telegram-bot<20.0.0
-    - arrow
-    - cachetools
-    - requests
-    - urllib3
-    - jsonschema
-    - TA-Lib
-    - tabulate
-    - jinja2
-    - blosc
-    - sdnotify
-    - fastapi
-    - uvicorn
-    - pyjwt
-    - aiofiles
-    - psutil
-    - colorama
-    - questionary
-    - prompt-toolkit
-    - schedule
-    - python-dateutil
-    - joblib
-    - pyarrow
-
-
-    # ============================
-    # 2/4 req dev
-
-    - coveralls
-    - flake8
-    - mypy
-    - pytest
-    - pytest-asyncio
-    - pytest-cov
-    - pytest-mock
-    - isort
-    - nbconvert
-
-    # ============================
-    # 3/4 req hyperopt
-
-    - scipy
-    - scikit-learn
-    - filelock
-    - scikit-optimize
-    - progressbar2
-    # ============================
-    # 4/4 req plot
-
-    - plotly
-    - jupyter
-
-    - pip:
-        - pycoingecko
-        # - py_find_1st
-        - tables
-        - pytest-random-order
-        - ccxt
-        - flake8-tidy-imports
-        - -e .
-    # - python-rapidjso
+  - python>=3.8,<=3.10
+  - pip
+  - libta-lib=0.4.0 # this is required for ta-lib from requirements.txt
```

Comment:
[Conda forge](https://anaconda.org/conda-forge/libta-lib) has basically no information about this package.
If we assume this is an official ta-lib library - then i'd expect a link to it from the ta-lib documentation ... 

I'll not be accepting unknown dependencies (from unknown, potentially malicious sources).

Reply:
1. I noticed that the `libta-lib` library appears to have come from the same source as the popular `ta-lib` package from conda-forge. It's interesting to note that `libta-lib` has twice the number of downloads as `ta-lib`, and both packages seem to be quite popular.
2. While it's not entirely clear what you mean by "unknown source," it's worth noting that some of the packages used already in `requirements.txt` might be considered as such and possess similar risks, which is very hard to mitigate as manual analysis is quite tedious. A good practice might be to rely on automated security checkers of dependencies such as Safety or Dependabot from Github, which seems to be already used here.
3. It's also worth noting that the ta-lib library, which is already included in requirements.txt, has a link to this conda-forge library.

Reply:
for every package in `requirements.txt` - there's a corresponding pypi site - which links to the original source.
For all smaller packages (not major ones like pandas/numpy, ...) - we did this analysis before adding them (is it a popular package / still actively maintained).
now these things change and we don't look at this on an ongoing basis - but pypi makes this far easier than conda.


maybe it's "lack of experience with conda" - but i don't find this information for conda packages - all conda-forge packages seem to be "magically there" (without description) - without obvious link to the actual source project. so if i'm the first to publish a package on conda-forge - it's name is mine.